core/services/s4/orm.go

```diff
@@ -25,6 +25,7 @@ type SnapshotRow struct {
 	Version    uint64
 	Expiration int64
 	Confirmed  bool
+	Payload    []byte
```

Comment:
We can't include the whole payload in the snapshot. That will make it way too big.

Reply:
👍🏼 makes sense, will only fetch the size of the payload with `octet_length(payload)`.

I've evaluated also using `pg_column_size(payload)` which reflects the actual space disk by including the field size + metadata overhead + padding. But this would difficult testing given that the len(payload) we are doing won't be able to take this ~1byte extra into account.